app/controllers/event_steps_controller.rb

```diff
@@ -1,9 +1,11 @@
 class EventStepsController < ApplicationController
-  before_action :load_event
+  before_action :load_event 
```

Comment:
Please enabled `.editor-config` support in your editor - it should then start stripping white space at the end of lines